Unity-Technologies/Graphics
# Hdrp/dlss fixes for particles half res

### Purpose of this PR

The problem & Reason:
During Dynamic Resolution Scaling, half res transparency passes fall too much behind detail. This causes particles to look very aliased and other transparent geo look wrong. This is because we are downscaling half resolution, which gets rid of more detail.

The solution is to clamp how low we go on resolution (everything relative to the native resolution of course). 

This PR has a few improvements for DRS and low resolution transparency (UX and functionality as well):

1) Showing an info box in the material inspector if low resolution transparency is disabled in the current HDRP asset. This should make it easier for our users to find it.
![image](https://user-images.githubusercontent.com/789671/120514275-1aafad00-c39b-11eb-810e-6753b351fe80.png)

2) Added new checkboxes  / low res transparency min threshold when DRS is enabled. This will let users threshold low res transparency when the current frame scale falls below a certain threshold. Included here also a similar info box to warn the user.
![image](https://user-images.githubusercontent.com/789671/120514459-4df23c00-c39b-11eb-9ef1-03ca326909d7.png)

3) Documentation update, and CHANGELOG updates.


---
### Testing status
In Windows dx11, just created a new material with low res transparent. Proceeded to test:
* Live enabling and disabling low res transparent from the HDRP asset. Checked that both info boxes update appropiately.
* Set DRS as force resolution percentage, and set many resolutions.
* Set the new threshold at multiple settings and verified that low res transparent was always disabled when the forced percentage fell between a threshold.
* Verified low transparency was running during these tests, and also verified it was disabled by simply looking at the low res result.
* Tested this on CAS and Bilinear, software DRS.

Recommended additional tests for QA (have not tested):
* Test on DLSS dx12 and vulkan.
* Test on hardware DRS
* Test on spaceship demo, with DLSS and verify particles can pop in / out of low res.
